hotdoc/utils/utils.py

```diff
@@ -204,16 +206,17 @@ def get_extension_classes(sort: bool,
     all_classes = {}
     deps_map = {}
 
-    for entry_point in pkg_resources.iter_entry_points(
+    for entry_point in entry_points(
             group='hotdoc.extensions', name='get_extension_classes'):
-        if entry_point.module_name == 'hotdoc_c_extension.extensions':
-            continue
         try:
+            if entry_point.module == 'hotdoc_c_extension.extensions':
+                continue
+            ep_name = entry_point.name
```

Comment:
Same validation issue here; since we can't be sure `entry_point` _has_ a `.module` property, we need to check it inside the `try:`.